examples/map.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/AlecAivazis/survey"
+)
+
+// the questions to ask
+var simpleQs = []*survey.Question{
+	{
+		Name: "name",
+		Prompt: &survey.Input{
+			Message: "What is your name?",
+		},
+		Validate: survey.Required,
+	},
+	{
+		Name: "color",
+		Prompt: &survey.Select{
+			Message: "Choose a color:",
+			Options: []string{"red", "blue", "green"},
+		},
+		Validate: survey.Required,
+	},
+}
+
+func main() {
```

Comment:
Alright I was skeptical at first you make a good point. You've convinced me supporting map[string] interface{} does make sense since it's relatively straight forward but id like to avoid the string assertion PR since that seems to be a bigger change that I want to think about a bit deeper. I'll go through this PR closely today and will let u know if there's anything else that needs changing.